api/src/pcapi/core/mails/transactional/educational/eac_finance_incident_notification.py

```diff
@@ -0,0 +1,31 @@
+from pcapi.core import mails
+from pcapi.core.finance import models as finance_models
+from pcapi.core.mails import models as mails_models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+
+
+def send_eac_finance_incident_email(
+    finance_incident: finance_models.FinanceIncident,
+) -> bool:
+    offer_names = {
+        booking_finance_incident.collectiveBooking.collectiveStock.collectiveOffer.name
+        for booking_finance_incident in finance_incident.booking_finance_incidents
+    }
+    venue = finance_incident.venue
+
+    if not venue.current_reimbursement_point or not venue.current_reimbursement_point.bookingEmail or not offer_names:
+        return True
+
+    result = True
+    for offer_name in offer_names:
+        data = mails_models.TransactionalEmailData(
+            template=TransactionalEmail.RETRIEVE_INCIDENT_AMOUNT_ON_COLLECTIVE_BOOKINGS.value,
+            params={"OFFER_NAME": offer_name, "VENUE_NAME": venue.common_name},
+        )
+
+        result &= mails.send(
```

Comment:
Pourquoi utiliser `&=` ? Première fois que je le vois

Reply:
Pour que la fonction renvoie `False` si au moins une demande d'envoi a échoué.
Ça revient au même que : `result = result and mails.send(`